pool.go

```diff
@@ -259,11 +259,12 @@ func (p *pool) checkPool() {
 					return
 				}
 				p.lk.RUnlock()
-				trialTimeout, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+				trialTimeout, cancel := context.WithTimeout(context.Background(), DefaultSaturnMirrorRequestTimeout)
```

Comment:
Tracked down high mirrored request failure rate to the really low timeout of 30 seconds which wont work for fetching entire CARs.  We should use atleast 1/3rd of the actual timeout for CAR requests (now that we make mirrored requests in parallel, this should be fine.)